specification/dimensions/chargedescription.md

```diff
@@ -0,0 +1,30 @@
+# Charge Description
+
+A Charge Description provides a high-level context of a line item without requiring additional discovery.  This column is typically a brief, human-readable summary of a select group of corresponding details across a cost and usage dataset or provides information not otherwise available.
+
+The Charge Description column MUST be present within billing data, MUST be of type String, and SHOULD not be null.
```

Comment:
[RFC 2119](https://www.rfc-editor.org/rfc/rfc2119.html) says this should be "SHOULD NOT"